imod/tests/test_typing/test_typing_grid.py

```diff
@@ -64,3 +69,56 @@ def test_enforce_dim_order__unstructured(basic_unstructured_dis):
 
     assert actual.dims == ibound.dims
     assert isinstance(actual, type(ibound))
+
+
+def test_is_transient(basic_dis, basic_unstructured_dis):
```

Comment:
I don't think this name is appropriate: You don't seem to call ``is_transient`` here, I only see a call to ``enforce_dim_order`` 

Some leftover copypasta?

Reply:
indeed, I removed it.